templates/partials/topnav.html

```diff
@@ -18,5 +21,141 @@
             </div>
         </li>
         {%- endfor -%}
+        <li class="imenu__list__item imenu__list__item--more">
+            <div class="imenu__list__anchor-container">
+                <button type="button" class="imenu__list__anchor" aria-expanded="false">
+                    Mer<span class="imenu__list__anchor-span"></span>
+                    <svg class="imenu__list__anchor-icon-right icon" focusable="false" aria-hidden>
+                        <use xlink:href="#docs-icon-arrow-down"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="ipopupmenu ipopupmenu--vertical">
+                <ul class="ipopupmenu__list ipopupmenu__list--more">
+                    <!-- prettier-ignore -->
+                    {%- for nav in topnav.children -%}
+                    {% set navPath = nav.path | replace("html", "") %}
+                    {% set navPath = navPath | replace(".", "") %}
+                    {% set activePage = navPath in doc.fileInfo.fullPath %}
+                    <li
+                        class="ipopupmenu__list__item {% if activePage %}ipopupmenu__list__item--highlight{% endif %}"
+                        data-item="nav-item-{{loop.index}}"
+                    >
+                        <a href="{{ nav.path | relative(doc) }}"> {{ nav.title }} </a>
+                    </li>
+                    {%- endfor -%}
+                </ul>
+            </div>
+        </li>
     </ul>
 </nav>
+
+<script>
+    function getElementsRefs() {
+        const nav = document.querySelector("#topnav");
+        const moreItem = nav.querySelector(".imenu__list__item--more");
+        const moreButton = moreItem.querySelector("button");
+        const popoverContainer = moreItem.querySelector(".ipopupmenu");
+        const popover = popoverContainer.querySelector(".ipopupmenu__list");
+        const menuItems = Array.from(nav.querySelectorAll(".imenu__list__item[data-item]"));
+        const popoverItems = Array.from(popover.querySelectorAll(".ipopupmenu__list__item[data-item]"));
```

Comment:
```suggestion
        const popoverItems = Array.from(popover.querySelectorAll("li"));
```

Is it not enough to select the `<li>` elements here? Makes for an easier-to-read selector.

Reply:
I don't want to include the more item.

- [ ] simplify query

Reply:
Fixed in ca30434821df95859e9b535e0e440e9e4ad0bbf6.